astroid/nodes/scoped_nodes/utils.py

```diff
@@ -0,0 +1,35 @@
+# Licensed under the LGPL: https://www.gnu.org/licenses/old-licenses/lgpl-2.1.en.html
+# For details: https://github.com/PyCQA/astroid/blob/main/LICENSE
+
+"""
+This module contains utility functions for scoped nodes.
+"""
+
+import builtins
+from typing import TYPE_CHECKING, List, Tuple
+
+from astroid.manager import AstroidManager
+
+if TYPE_CHECKING:
+    from astroid import nodes
+
+
+_builtin_astroid: "nodes.Module | None" = None
```

Comment:
I've renamed it to lowercase, only constants should be uppercase, and added an underscore since it's only used in this module.

Reply:
Thanks! That was quite obvious.